core/libs/editor/src/lib/editor.service.ts

```diff
@@ -260,29 +272,32 @@ export class EditorService {
   }
 
   private validateCurrentModel(rdfModel: RdfModel) {
-    return this.getIdentifiedModelValidator(rdfModel).pipe(
-      tap((validationResults: Array<any>) => {
+    const serializedModel = this.rdfService.serializeModel(rdfModel);
+
+    return this.modelApiService.getValidationErrors(serializedModel).pipe(
+      tap((validationResults: Array<SemanticError | SyntacticError | ProcessingError>) => {
         if (this.modelValidatorService.checkForCriticalErrors(validationResults, rdfModel)) {
-          throwError(() => null);
+          throwError(null);
```

Comment:
Please test it with a case of error to see if `throwError` will do something. I am not sure it will function in this circumstances

Reply:
Method will be removed because it is no longer used.